acados/ocp_nlp/ocp_nlp_cost_ls.c

```diff
@@ -179,6 +179,27 @@ void ocp_nlp_cost_ls_dims_set(void *config_, void *dims_, const char *field, int
 
 
 
+/* dimension getters */
+static void ocp_nlp_cost_ls_get_ny(void *config_, void *dims_, int* value)
+{
+    ocp_nlp_cost_ls_dims *dims = (ocp_nlp_cost_ls_dims *) dims_;
+    *value = dims->ny;
+}
+
+
+
+void ocp_nlp_cost_ls_dims_get(void *config_, void *dims_, const char *field, int* value)
+{
+    if (!strcmp(field, "ny"))
+    {
+        ocp_nlp_cost_ls_get_ny(config_, dims_, value);
+    }
+    else
+    {
+        printf("error: attempt to get dimension from constraint model, that is not there");
```

Comment:
I would go for an error message like: 
`printf("error: ocp_nlp_cost_ls_dims_get: wrong field %s\n", field);`
which is much better for debugging, because it tells you exactly where and what is the issue

Reply:
good point, I must have copy pasted this from somewhere else. For now I have fixed these ones (see a90f6f8)